package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-header",
   "description": "Fozzie Header – Header Component for Just Eat projects",
-  "version": "0.11.1",
+  "version": "0.12.1",
```

Comment:
The version would be `0.12.0` – once the minor version increments it sets the patch back to zero

Reply:
This now needs to be v0.13.0 as a new release has been published